src/GafferUIModule/PathListingWidgetBinding.cpp

```diff
@@ -1761,20 +1794,32 @@ IECore::PathMatcher getExpansion( uint64_t treeViewAddress )
 	return model ? model->getExpansion() : IECore::PathMatcher();
 }
 
-void setSelection( uint64_t treeViewAddress, const IECore::PathMatcher &paths, bool scrollToFirst, bool expandNonLeaf )
+void setSelection( uint64_t treeViewAddress, object pythonPaths, bool scrollToFirst, bool expandNonLeaf )
 {
-	IECorePython::ScopedGILRelease gilRelease;
-
 	QTreeView *treeView = reinterpret_cast<QTreeView *>( treeViewAddress );
 	PathModel *model = dynamic_cast<PathModel *>( treeView->model() );
+
+	PathModel::Selection paths;
+	boost::python::container_utils::extend_container( paths, pythonPaths );
+
+	IECorePython::ScopedGILRelease gilRelease;
 	model->setSelection( paths, scrollToFirst, expandNonLeaf );
 }
 
-IECore::PathMatcher getSelection( uint64_t treeViewAddress )
+list getSelection( uint64_t treeViewAddress )
 {
 	QTreeView *treeView = reinterpret_cast<QTreeView *>( treeViewAddress );
 	PathModel *model = dynamic_cast<PathModel *>( treeView->model() );
-	return model->getSelection();
+
+	list result;
+
+	PathModel::Selection selection = model->getSelection();
```

Comment:
Is there a reason we're taking a copy here? Might it be cheaper and clearer just to do `for( const auto &p : model->getSelection() )`?

Reply:
Nope, fixed in 60336bfa71fa897f245bdb9517d56294bfc4d5cc